Libraries/Image/RCTNetworkImageView.m

```diff
@@ -58,8 +74,35 @@ - (void)setImageURL:(NSURL *)imageURL resetToDefaultImageWhileLoading:(BOOL)rese
       self.layer.minificationFilter = kCAFilterTrilinear;
       self.layer.magnificationFilter = kCAFilterTrilinear;
     }
+    NSDictionary *event = @{
```

Comment:
Sorry, XCode makes me crazy about formatting thinks like this. Used this https://github.com/github/objective-c-style-guide#literals